src/item.cpp

```diff
@@ -1594,7 +1595,7 @@ void item::basic_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
         if( type->min_per > 0 ) {
             req.push_back( string_format( "%s %d", _( "perception" ), type->min_per ) );
         }
-        for( const std::pair<const skill_id, int> &sk : type->min_skills ) {
+        for( const std::pair<const skill_id, int> sk : sorted_lex( type->min_skills ) ) {
```

Comment:
I totally agree. I added the `sorted_lex` because bunch of tests currently rely on the specific order of returned entities and I wanted to limit the scope of changes in this PR.